babel.config.js

```diff
@@ -5,6 +5,11 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+const semver = require('semver');
+const pkg = require('./package.json');
+
+const supportedNodeVersion = semver.minVersion(pkg.engines.node).version;
```

Comment:
preset-env doesn't support semver operators, so this is used to get the actual minimum version as a string (resolves to `8.3.0`)